src/envo/envo-edit.owl

```diff
@@ -3535,6 +3553,15 @@ Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510038>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510039>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510040>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510041>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510055>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510056>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510057>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510058>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510060>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510061>))
```

Comment:
>A slow drying ink which is made from refined soybean oil.

The "slow drying" is extraneous and messes up the definition pattern.
On the axioms, I don't think the pigments have to be biological - that axiom should be broadened